config/environments/development.rb

```diff
@@ -73,5 +73,5 @@
   # Allow incoming connections over ngrok
   config.hosts << /[a-z0-9]+\.ngrok\.io/
   config.offseason = false
-  config.ngrok_url = "http://cec1d9b787e4.ngrok.io"
+  config.ngrok_url = "http://d0d376ec6d64.ngrok.io"
```

Comment:
Consider setting up `development_overrides.rb` as a file that's ignored in `.gitignore`. Not a blocker.

Reply:
Gah i meant to leave this out!